go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/src-d/borges
```

Comment:
since it's an app and its dependencies are managed by dep, should we avoid for now having a go.mod too?